evap/staff/importers.py

```diff
@@ -165,6 +165,13 @@ def check_user_data_sanity(self):
             except UserProfile.DoesNotExist:
                 # nothing to do here
                 pass
+                
+            try:
+                user_same_name = UserProfile.objects.get(first_name=user_data.first_name, last_name=user_data.last_name)
+                self.warnings.append(u"Warning: The user {} {} already exists.".format(user_same_name.first_name, user_same_name.last_name))
```

Comment:
please wrap the string in _("string goes here"), this tells django to translate it.

also, @janno42 and i discussed this and we think we need the full data of the existing and new user.

something like
"Warning: The existing user {} ({} {} {}, {}) has the same first and last name as the following user from the enrollment data: {} ({} {} {}, {})". you could copypaste that structure from a few lines above. if you have an idea how to not duplicate the code, please do that as well ;)